hedera-node/hedera-app/src/main/java/com/hedera/node/app/roster/RosterServiceImpl.java

```diff
@@ -29,13 +30,10 @@
  * Not exposed outside `hedera-app`.
  */
 public class RosterServiceImpl implements Service {
-    /** The name of this service */
-    public static final String NAME = "RosterService";
-
     @NonNull
     @Override
     public String getServiceName() {
-        return NAME;
+        return RosterStateId.NAME;
```

Comment:
nit: Do we need a new class to define these Constants. Are we expecting to add more information to that class ?

Reply:
This was done to address a comment above in this PR. Please see the thread at: https://github.com/hashgraph/hedera-services/pull/15096#discussion_r1731856354